pkg/porter/create.go

```diff
@@ -1,43 +1,73 @@
 package porter
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create function creates a porter configuration in the specified directory (bundleName) or in the current directory if no directory is specified.
+// The bundle's name will be the given bundleName, or "porter_hello" if bundleName is the empty string
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// If given a bundleName, create directory if it doesn't exist
+	if bundleName != "" {
+		_, err := os.Stat(bundleName)
+		if err != nil && errors.Is(err, os.ErrNotExist) {
+			err = os.Mkdir(bundleName, os.ModePerm)
```

Comment:
Troy wanted a MkdirAll here, but I think it makes more sense with a Mkdir because otherwise the bundlename could contain slashes (/) which I'm not sure is what we want.

Reply:
The MkDirAll would make the bundle at the path defined but not name the bundle the whole path. 
I should be able to make a bundle at `really/cool/path/awesome` and the bundle should be awesome but the path and scaffold should be at the path.